text/0107-construct-library-module-lifecycle.md

```diff
@@ -0,0 +1,146 @@
+
+While experimenting on the module API design and implementation, we will be as transparent as possible with customers about the work in progress. To achieve this we will use the module’s GitHub tracking issue to document use cases and construct design as they are developed in each iteration. Not all communication is required to occur in the tracking issue itself. For large and/or complex changes, we will author a Request for Comments (RFC) that outlines our design and publish it on the roadmap for feedback. We will also use pull requests to conduct construct reviews. In all cases, relevant RFCs, PRs, bugs, and feature requests that are in scope are linked in the module’s tracking issue for visibility. See [Appendix A](#Appendix-A:-GitHub-Artifacts) for an illustration of how these GitHub artifacts are used and related to each other.
+
+* **Stage 1 activities:**
+
+    * **Identify key stakeholders** - Opportunistically recruit stakeholders outside the CDK team to participate in API reviews and usability studies. Potential stakeholders include: service's product manager, service's developer advocates (if any), service's software development engineer(s) (designated by service's leadership), solutions architects who specialize in service (sourced via internal mailing lists), and AWS Developer Tools developer advocates. 
+    * **Update GitHub tracking issue** - Summarize the primary use cases and required constructs as they are identified
+    * **Author RFCs [optional]** - Request for comment documents are created for large / complex designs that require explanation and robust discussion. RFCs are stored in a separate [GitHub repo](https://github.com/aws/aws-cdk-rfcs), and the module's tracking issue will include links to relevant RFCs.
+    * **Update Roadmap** - publicly indicate the module is work in progress
+        * If an RFC is written, place the module’s tracking issue on the CDK roadmap in the “researching” column to indicate we need customer feedback on the module’s design
+        * otherwise, place the module’s tracking issue on the CDK roadmap in the “Working on it“ column
+    * **Submit Pull Requests** - pull requests implementing the constructs are submitted, the API design reviewed (by module owner), and merged
+    * **Documentation** - update module’s README with explanations and code snippets that illustrate how to use new constructs as they are developed
+    * **Publish aws-xyz-experimental package** - deliver experimental constructs in their own package to clearly communicate module maturity. This package is tagged with Stability:Experimental
+    * **Observe** - monitor module utilization and GitHub issues filed against the experimental module
```

Comment:
"monitor module utilization" -- same comment as the previous section; not clear who is monitoring it, how and what are they looking for?

Reply:
Will, clarify.